### PR TITLE
ipatests: Test AES SHA 256 and 384 Kerberos enctypes enabled

### PR DESCRIPTION
**Pull request by mrizwan93**
AES SHA 256 and 384-bit enctypes supported by MIT kerberos but
was not enabled in IPA. This test is to check if these types are
enabled.

related: https://pagure.io/freeipa/issue/8110

Signed-off-by: Mohammad Rizwan Yusuf <myusuf@redhat.com>

### CODE REVIEW

**Comment by fcami** [comment]
Hi @mrizwan93 could you please explain why this test case should go into this particular class? TestInstallWithCA_KRA2 executes the tests defined in InstallTestBase2 so far.

**Comment by mrizwan93** [comment]
@fcami moved to `TestInstallMaster`

**Comment by fcami** [comment]
Hi @mrizwan93 I still fail to see how this test is installation-related and I see including this in any non-related test suite quite confusion. Maybe a new test suite test_sanity would do.

**Comment by mrizwan93** [comment]
HI @fcami , Sorry I didn't got your point in previous iteration. 

The test is to check if  `aes128-sha2:normal`  `aes128-sha2:special`  `aes256-sha2:normal` and `aes256-sha2:special` are included in `krbSupportedEncSaltTypes` on newly installed master. IMO, it is relevant to this suite.

**Comment by fcami** [comment]
hi @mrizwan93 It's not exactly installation-related, more like features-related.
We have a [test suite](https://github.com/freeipa/freeipa/blob/master/ipatests/test_integration/test_commands.py#L120) where this would fit. Could you please move the test to that? Thanks.

File: ipatests/test_integration/test_commands.py [hunk 1]
```diff
@@ -130,6 +131,27 @@ def get_cert_base64(self, host, path):
...
+        AES SHA 256 and 384-bit enctypes supported by MIT kerberos but
+        was not enabled in IPA. This test is to check if these types are
+        enabled.
+
+        related: https://pagure.io/freeipa/issue/8110
+        """
+        tasks.kinit_admin(self.master)
+        realm = self.master.domain.realm
+        suffix = realm_to_suffix(realm)
+        dn = DN(("cn", realm), ("cn", "kerberos")) + suffix
```
Comment (rcritten) [inline ipatests/test_integration/test_commands.py]: IMHO the extra variables aren't needed and you don't need to add the DNs:

dn = DN(("cn", realm), ("cn", "kerberos"), realm_to_suffix(self.master.domain.realm))

**Comment by rcritten** [comment]
Please remove the temp commit.

**Comment by rcritten** [comment]
master:

* b0d57d99e5719099ed0be0476985ab394b131cb0 Test AES SHA 256 and 384 Kerberos enctypes enabled

**Pull request closed without merging**